### ISSUE TITLE
i18n script should support path option

### ISSUE THREAD

**Issue by ericof** [plone/volto#5062]
**Is your feature request related to a problem? Please describe.**

Currently, it is not possible to run the i18n script and update the locales in a project containing addons in src/addons/

**Describe the solution you'd like**

The script should either have the option to update the locales for an addon in a project

**Describe alternatives you've considered**

It is possible to get the same result by:

1. Copying the `babel.config.js` to the addon root (i.e. src/addons/volto-addon/)
2. `cd src/addons/volto-addon/ && ../../node_modules/.bin/i18n --addon`

**Comment by davisagli** [comment]
@sneridagh doesn't that require a babel configuration at the root of the addon repo?

**Comment by sneridagh** [comment]
@davisagli yes, we were already doing this, it's on the generator since long ago: https://github.com/plone/volto/blob/master/packages/generator-volto/generators/addon/templates/babel.config.js

but it's true that we were removing it in the docker implementations because we relied that the docker approach would provide everything needed.

Even if we come up with a script that does it, it will be needed anyways...

**Comment by sneridagh** [comment]
I made good progress using trying

`process.chdir('src/addons/my-addon')`

, then run the original script, still some rough edges... and we will have to get back the babel config.

**Comment by davisagli** [comment]
@sneridagh well if it's ok to add the babel config back for the docker approach, then we don't need this. possibly `make i18n` in the docker setup could even temporarily copy the babel config and leave it out of the repo.

**Comment by sneridagh** [comment]
@davisagli since it's something that will work even without the dockerized approach, I would include it anyways. In fact, the generator has it since some time ago, so it's already there.

**Comment by davisagli** [comment]
@sneridagh ok, then I think we can close this, and we just have to re-add the babel config to the addons where we are using the dockerized approach. @ericof do you agree?

**Comment by davisagli** [comment]
This was about the old generator projects, we have a consistent setup with cookieplone now.
